index.html

```diff
@@ -805,10 +805,10 @@ <h5 property="schema:name"><span>Note</span>: HTTP Method and Access Mode Mappin
                             <dd>The HTTP <code>POST</code> can be used to create a new resource in a container or add information to existing resources (but not remove resources or its contents) with either <code>acl:Append</code> or <code>acl:Write</code>.</dd>
 
                             <dt id="http-put"><code>PUT</code></dt>
-                            <dd>As the HTTP <code>PUT</code> method requests to create or replace the resource state, the <code>acl:Write</code> access mode is required to replace an existing resource, but to create a new resource, access to the container can be either <code>acl:Append</code> or <code>acl:Write</code>.</dd>
+                            <dd>As the HTTP <code>PUT</code> method requests to create or replace the resource state, the <code>acl:Write</code> access mode is required to replace an existing resource, but to create a new resource, access to the container additionally needs to be either <code>acl:Append</code> or <code>acl:Write</code>.</dd>
```

Comment:
* "the requester" needing particular access is not necessarily true or at least is subject to misinterpretation. I'd suggest to instead keep focus on the modes as this non-normative section is specifically about mapping the HTTP request semantics to access modes.
* "and/or" can be simplified to "or" as the the latter inherently covers both being true. So, I suggest to keep the original text
* `<i>` is not needed. No objection to leaving it in.

Reply:
* I've removed `the requester`.
* `"and/or" can be simplified to "or"` — Technically true. However, many people read `or` as `xor` rather than as the intended `and/or`. This is why I try to always write `and/or` when that is the intended meaning.
* `<i>` is not *needed*. To me, the extra emphasis on the desired action helps to make the separation of required access mode(s) clearer. I will live without it if need be.
```suggestion
                            <dd>The HTTP <code>PUT</code> method requests to create or replace the resource state. <i>Creating</i> a new resource requires <code>acl:Append</code> and/or <code>acl:Write</code> access to the container as well as <code>acl:Write</code> access to the (new) resource. <i>Replacing</i> an existing resource requires only <code>acl:Write</code> access to the resource itself.</dd>
```